### PR TITLE
add retries to apt and packer

### PR DESCRIPTION
**Pull request by Rozzii**
This commit adds retires to apt and packer commands involved in openstack,jumphost,base and node image building.

### CODE REVIEW

File: ci/scripts/image_scripts/setup_chrony_ubuntu.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #! /usr/bin/env bash
 
+# Set apt retry limit to higher than default to
+# make the data retrival more reliable
+sudo echo 'Acquire::Retries "10";' > /etc/apt/apt.conf.d/80-retries
```
Comment (tuminoid) [inline ci/scripts/image_scripts/setup_chrony_ubuntu.sh:5]: Don't put any code above `set -eu` as it might fail and set wouldn't catch it.

**Review by tuminoid — CHANGES_REQUESTED** [review]

File: ci/scripts/image_scripts/setup_chrony_ubuntu.sh (hunk 1 above)
Comment (Rozzii) [reply ci/scripts/image_scripts/setup_chrony_ubuntu.sh:5]: there is no set -eu in the script

Comment (Rozzii) [reply ci/scripts/image_scripts/setup_chrony_ubuntu.sh:6]: done

**Comment by Rozzii** [comment]
> Do we really need to put the apt config in all these scripts? Is it not enough to put it in `provision_*.sh` scripts that then call the other scripts?

I was not sure about all teh use-cases of all the script, just wanted to be sure that the retries are there, if you think it is not needed everywhere i can put it only to provision scripts

**Review by tuminoid — APPROVED** [review]
lgtm

**Comment by lentzi90** [comment]
> > Do we really need to put the apt config in all these scripts? Is it not enough to put it in `provision_*.sh` scripts that then call the other scripts?
> 
> I was not sure about all the use-cases of all the scripts, just wanted to be sure that the retries are there, if you think it is not needed everywhere i can put it only to provision scripts

I'm fairly certain that they are only ever used by the provision scripts. Did a quick search through the repo for any references and could not find any other than those scripts.
Strictly speaking the metal3 and jenkins images are also built on the base image so we would only need it in base and node provision scripts, but that could change in the future. I would anyway suggest to add it only in the provision scripts.

**Comment by kashifest** [comment]
> /retest

I am fixing the shellcheck issues in my new PR. It wont pass until that merges

**Review by kashifest — APPROVED** [review]

**Pull request merged**